File: src/main/java/namingservice/replies/Reply.java
package namingservice.replies;

import com.google.gson.Gson;

public class Reply {
    private String reply_type;
    private boolean request_resolved;

    public Reply(String reply_type, boolean request_resolved) {
        this.reply_type = reply_type;
        this.request_resolved = request_resolved;
    }

    public String getReply_type() {
        return reply_type;
    }

    public void setReply_type(String reply_type) {
        this.reply_type = reply_type;
    }

    public boolean isRequest_resolved() {
        return request_resolved;
    }

    public void setRequest_resolved(boolean request_resolved) {
        this.request_resolved = request_resolved;
    }

    public byte[] getBytes(){
        return new Gson().toJson(this).getBytes();
    }
}

File: ObjectRequestBrokerCS/ObjectRequestBrokerCS/namingservice/replies/Reply.cs
namespace namingservice.replies
{

	public class Reply
	{
		private string reply_type;
		private bool request_resolved;

		public Reply(string reply_type, bool request_resolved)
		{
			this.reply_type = reply_type;
			this.request_resolved = request_resolved;
		}

		public Reply()
		{
		}

		public virtual string Reply_type
		{
			get
			{
				return reply_type;
			}
		}

		public virtual bool Request_resolved
		{
			get
			{
				return request_resolved;
			}
		}
	}

}